src/hooks/common/use-bind-pointer-movement-event-on-window.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { useEffect } from 'react'
+
+/**
+ * Registers event listener for pointer movement and pointer release on the window object.
+ *
+ * @param onPointerMovement is triggered if the user moves the pointer over the window
+ * @param onPointerRelease is triggered if the pointer is released (touch end or mouse up)
+ */
+export const useBindPointerMovementEventOnWindow = (
+  onPointerMovement: (event: MouseEvent | TouchEvent) => void,
+  onPointerRelease: () => void
+) => {
+  useEffect(() => {
+    const pointerMovement = onPointerMovement
+    const pointerRelease = onPointerRelease
```

Comment:
Why the reassignement?

Reply:
This is necessary for the effect. Remove event listener needs EXACTLY the function that was bound before. So i need to save this function here to remove it in the effect return function.